### PR TITLE
Update dual-stack validation for 1.20

### PR DESCRIPTION
**Pull request by lachie83**
Co-authored-by: Bridget Kromhout <bridget@kromhout.org>

Signed-off-by: Lachlan Evenson <lachlan.evenson@microsoft.com>

### CODE REVIEW

File: content/en/docs/tasks/network/validate-dual-stack.md [hunk 1]
```diff
@@ -134,28 +137,46 @@ status:
...
 
-{{< codenew file="service/networking/dual-stack-ipv6-svc.yaml" >}}
+{{< codenew file="service/networking/dual-stack-preferred-svc.yaml" >}}
 
-Validate that the Service gets a cluster IP address from the IPv6 address block. You may then validate access to the service via the IP and port.
-```
- kubectl get svc -l app=MyApp
-NAME         TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)   AGE
-my-service   ClusterIP   fe80:20d::d06b   <none>        80/TCP    9s
+Validate that the Service gets a cluster IPs from the IPv4 and IPv6 address blocks using `kubectl describe`. You may then validate access to the service via the IPs and ports.
```
Comment (bridgetkromhout) [inline content/en/docs/tasks/network/validate-dual-stack.md]: `a cluster IPs`

As you are mentioning IPv4 and IPv6 I think you probably want `cluster IPs` (plural, no `a`).

File: content/en/docs/tasks/network/validate-dual-stack.md [hunk 2]
```diff
@@ -134,28 +137,46 @@ status:
...
+
+{{< note >}}
+The `kubectl get svc` command will only show the primary IP in the `CLUSTER-IP` field.
```
Comment (bridgetkromhout) [inline content/en/docs/tasks/network/validate-dual-stack.md:184]: It's good to have this explanation, but should we also show the output of something like ` kubectl get svc -l app=MyApp` for additional clarity?

Comment (lachie83) [reply content/en/docs/tasks/network/validate-dual-stack.md:184]: Added

File: content/en/docs/tasks/network/validate-dual-stack.md (hunk 1 above)
Comment (lachie83) [reply content/en/docs/tasks/network/validate-dual-stack.md]: Fixed

File: content/id/examples/service/networking/dual-stack-preferred-svc.yaml [hunk 3]
```diff
@@ -3,10 +3,10 @@ kind: Service
...
-  ipFamily: IPv6
+  ipFamilyPolicy: PreferDualStack
   selector:
     app: MyApp
   ports:
     - protocol: TCP
       port: 80
-      targetPort: 9376
\ No newline at end of file
+      targetPort: 9376
```
Comment (bridgetkromhout) [inline content/id/examples/service/networking/dual-stack-preferred-svc.yaml]: So do we still want this example port? It adds possible ambiguity.

Comment (lachie83) [reply content/id/examples/service/networking/dual-stack-preferred-svc.yaml]: Yes we should remove any fields that aren't necessary.

**Comment by bridgetkromhout** [comment]
LGTM.

**Pull request merged**